### PR TITLE
MAGENTO-2002: fixed invoicing braintree orders

### PR DESCRIPTION
**Pull request by andre-kraus**

### CODE REVIEW

File: src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php [hunk 1]
```diff
@@ -23,8 +23,9 @@ class Shopgate_Framework_Model_Payment_Braintr_Abstract
     extends Shopgate_Framework_Model_Payment_Abstract
     implements Shopgate_Framework_Model_Payment_Interface
 {
-    const ADVANCED_FRAUD_REVIEW  = 'Review';
-    const ADVANCED_FRAUD_DECLINE = 'Decline';
+	const ADVANCED_FRAUD_REVIEW              = 'Review';
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: I don't believe we have `=` alignments in framework, the tab characters are also used here?

File: src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php [hunk 2]
```diff
@@ -203,10 +204,12 @@ protected function _addBraintreeInfo($payment, $braintreeId, $riskDataId)
      */
     protected function _addInvoice($paymentInfo)
     {
-        if (!$this->getShopgateOrder()->getIsPaid()) {
+	    if (!$this->getShopgateOrder()->getIsPaid()
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: tab chars

**Review by augsteyer — CHANGES_REQUESTED** [review]

File: src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php (hunk 2 above)
Comment (augsteyer) [reply src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: also remove the unnecessary return spacing

File: src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php (hunk 1 above)
Comment (andre-kraus) [reply src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: Actually we are aligning them in the code (most of the time), but not always for constants. I would recommend that we should do it all the times. Objections?

Comment (augsteyer) [reply src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: no objections here

**Comment by augsteyer** [comment]
these changes are fine, but please note that we have not tested `authorization` at all with braintree in ticket MAGENTO-1991, so this ticket may need more fixes

File: src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php (hunk 2 above)
Comment (andre-kraus) [reply src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php]: Fixed

**Review by augsteyer — APPROVED** [review]

**Pull request merged**